AsyncDisplayKit/Details/ASPINRemoteImageDownloader.h

```diff
@@ -20,8 +20,23 @@ NS_ASSUME_NONNULL_BEGIN
 
 @interface ASPINRemoteImageDownloader : NSObject <ASImageCacheProtocol, ASImageDownloaderProtocol>
 
+/**
+ * A shared image downloader which can be used by @c ASNetworkImageNodes and @c ASMultiplexImageNodes
+ *
+ * This is the default downloader used by network backed image nodes if PINRemoteImage and PINCache are
+ * available. It uses PINRemoteImage's features to provide caching and progressive image downloads.
+ */
 + (ASPINRemoteImageDownloader *)sharedDownloader;
 
+/**
+ * The shared instance of a @c PINRemoteImageManager used by all @c ASPINRemoteImageDownloaders
+ *
+ * @discussion you can use this method to access the shared downloader. This is useful to share a cache
```

Comment:
Also, "use this to access the -memoryCache or -diskCache objects and set parameters like their maximum size, as well as handle authentication challenges for image downloads"